contracts/ProtocolWallet.sol

```diff
@@ -0,0 +1,79 @@
+pragma solidity 0.5.16;
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "./spec_interfaces/IProtocolWallet.sol";
+import "./WithClaimableMigrationOwnership.sol";
+import "./WithClaimableFunctionalOwnership.sol";
+import "@openzeppelin/contracts/math/SafeMath.sol";
+
+contract ProtocolWallet is IProtocolWallet, WithClaimableMigrationOwnership, WithClaimableFunctionalOwnership{
+    using SafeMath for uint256;
+
+    IERC20 public token;
+    address public client;
+
+    uint lastApprovedAt;
+    uint annualRate;
+
+    modifier onlyClient() {
+        require(msg.sender == client, "caller is not the committee contract");
+
+        _;
+    }
+
+    constructor(IERC20 _token, address _client) public {
+        token = _token;
+        client = _client;
+        lastApprovedAt = now;
```

Comment:
Makes sense to init in the constructor, an alternative is to init on setMaxAnnualRate when set from 0 to != 0.
Need to verify that the client can operate correctly with 0 value (ERC20 does) to allow early deployment.